### PR TITLE
fix(.travis.yml): modify travis.yml file to be client-specific

### PR DESCRIPTION
**Pull request by arschles**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,25 @@
...
-go:
-  - 1.5.1
-env:
-  - GO15VENDOREXPERIMENT=1
+language: generic
 branches:
   only:
     - master
+env:
+  DEV_REGISTRY=quay.io
```
Comment (mboersma) [inline .travis.yml]: Does this need to be set for CI? The [`deploy.sh` script](https://github.com/deis/workflow-cli/blob/master/_scripts/deploy.sh#L10-L12) sets its own values for `DEV_REGISTRY` as needed.

Comment (mboersma) [inline .travis.yml]: Oh, I see--not used any more. Never mind.

File: .travis.yml [hunk 2]
```diff
@@ -1,48 +1,25 @@
...
-  - make test
-  - make -C client/ bootstrap build test dist prep-bintray-json
+  - make build test dist prep-bintray-json
 deploy:
-  - provider: script
+  - provider: bintray
     # ensure client/doc builds aren't removed
     # see https://docs.travis-ci.com/user/deployment/#Uploading-Files
     skip_cleanup: true
-    script: _scripts/deploy.sh
```
Comment (mboersma) [inline .travis.yml]: We should also remove `_scripts/deploy.sh` if it isn't used now.

Comment (arschles) [inline .travis.yml]: @mboersma good point - done in https://github.com/deis/workflow-cli/commit/cb8ee509338301f0b94a955eb0c4402393c1422c

File: .travis.yml (hunk 1 above)
Comment (arschles) [inline .travis.yml]: @mboersma good catch - removed in https://github.com/arschles/deis-workflow-cli/commit/3c52ee0011121d07831cc007faaf170b86bec1a5

**Pull request merged**